cr17/msg/scoopControl.msg

```diff
@@ -0,0 +1,3 @@
+float64 arm
+float64 scoop
+
```

Comment:
There should be a header to hold timestamp and frame id information to show when and where these angles have meaning.